plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/SystemVariablesResolver.java

```diff
@@ -41,6 +42,13 @@
     public static final String VAR_DBEAVER_HOME = "dbeaver_home";
     public static final String VAR_LOCAL_IP = "local.ip";
 
+    private Properties configurationFile = null;
+
+    public void setConfigurationFilePath(Properties configurationFile) {
```

Comment:
We're operating over properties, not paths. Please rename both field and method to something more meaningful.